js/app/core/data.service.js

```diff
@@ -0,0 +1,39 @@
+((app) => {
```

Comment:
Are we going to put ```"use strict";``` in angular modules?

Reply:
done

Reply:
i actually had a smaller pull request #23 but i had to keep working, so i branched of that request